sdk/cosmos/azure-cosmos-spark_3-1_2-12/src/main/scala/com/azure/cosmos/spark/CosmosConfig.scala

```diff
@@ -37,6 +37,7 @@ private object CosmosConfigNames {
   val PreferredRegionsList = "spark.cosmos.preferredRegionsList"
   val ApplicationName = "spark.cosmos.applicationName"
   val UseGatewayMode = "spark.cosmos.useGatewayMode"
+  val ReadCustomQuery = "spark.cosmos.read.customQuery"
```

Comment:
please also update the reference config documentation

Reply:
Done